docs/source/concepts/time.rst

```diff
@@ -11,7 +11,52 @@ Thinking about Time in the Simulation
 
 Outline
 -------
- - The simulation clock
+ - The Simulation Clock
  - Start, stop, and step size.
  - Fundamental assumptions about discrete time simulation.
- - builder.time.clock().clock() vs. event.time
+ - builder.time.clock() vs. event.time
+ - Individual Clocks
+
+The Simulation Clock
+--------------------
+The :class:`SimulationClock <vivarium.framework.time.SimulationClock>` plugin manages the progression of time throughout the simulation. 
+Fundamentally, that means it keeps track of the current time (beginning at the *start time*), provides
+a mechanism to advance the simulation time by some duration (the *step size*), and determines when 
+the simulation is complete via a configured *end time*. The simplest
+implementation of a Clock is the :class:`SimpleClock <vivarium.framework.time.SimpleClock>` object, which is little more
+than an integer counter that is incremented by a fixed step size until it reaches the
+end time. Public health scenarios rely on data that are often tied to particular years or rates, where the 
```

Comment:
We should probably not specifically call out the public health use case. Maybe better just to say something like "when modeling events in the real world it is often useful to use the DateTimeClock which ties simulation time to real dates".